index.html

```diff
@@ -102,37 +102,37 @@ <h1>Minecraft Bingo</h1>
 			<h4>Stats</h4>
-				<li>Goals that list stats like "Crouch a distance of 50 meters" make use of the Statistics screen in the Minecraft pause menu. You need to make it so that stat shows the amount (or greater).</li>
-				<li><em>Tip:</em> The Statistics screen is often slow to update, so leave time between checking it when you're trying to complete a goal.</li>
+				<li>Goals which list stats (such as "Crouch a distance of 50 meters") make use of the Statistics screen in the Minecraft pause menu. You need to ensure that stat shows the given amount (or greater).</li>
+				<li><em>Tip:</em> The Statistics screen often updates slowly, so leave some time between checking it when you're trying to complete a goal.</li>
 			<p></p>
-			<p>Create a new World in Minecraft and use <input class="seed_for_copying" id="seed_for_copying_howto" type="text" value="" size="6" readonly="readonly" /><button onclick='copySeedToClipboard("seed_for_copying_howto");'>Copy</button> as the Seed (this is the same random seed used to generate this sheet). Load the world so that it can generate but don't move until you're ready. Pause the game to make sure time of day doesn't change or you get attacked. When you're ready reveal the sheet if it's hidden and start playing!</p>
-			<p>If you're racing together with other players, generate a Bingo Sheet and then link it them so that everyone has the same goals. You can hide the table in the Options menu before generating the seed so that everyone can reveal sheet when you start. Make sure everyone uses the same Seed for the world too!</p>
-			<p><em>Tip:</em> You can click on the squares to change their colour which is useful for planning or to mark off completed goals. Use the colour slider in the options menu to choose many many colours you want to use. You can also hover over the square and press 0-6!</p>
+			<p>Create a new world in Minecraft and use <input class="seed_for_copying" id="seed_for_copying_howto" type="text" value="" size="6" readonly="readonly" /><button onclick='copySeedToClipboard("seed_for_copying_howto");'>Copy</button> as the Seed (this is the same random seed used to generate this sheet). Load the world so that it can generate but don't move until you're ready. Pause the game to make sure time of day doesn't change or you get attacked. When you're ready, reveal the sheet if it's hidden and start playing!</p>
+			<p>If you're racing together with other players, generate a Bingo sheet and then link it to them so that everyone has the same goals. You can hide the table in the Options menu before generating the seed so that everyone can reveal the sheet when you start. Make sure everyone uses the same Seed for the world too!</p>
+			<p><em>Tip:</em> You can click on the squares to change their colour which is useful for planning or to mark off completed goals. Use the colour slider in the Options menu to choose how many colours you want to use. You can also hover over the square and press (0-6)!</p>
```

Comment:
hover over the square and press (0-6)!

[HTML tag `<kbd>`](https://www.w3schools.com/tags/tag_kbd.asp) could be used here for `0` and `6`. Hyphen could be replaced by `&ndash;` (–).

    hover over the square and press <kbd>0</kbd>&ndash;<kbd>6</kbd>!

It will look approximately like this:

> hover over the square and press <kbd>0</kbd>&ndash;<kbd>6</kbd>!

Reply:
> It will look approximately like this

As it turns out, GitHub has quite a lot of CSS to make `<kbd>` tags look like that, so this assumption was incorrect. E.g. in Chrome `<kbd>` just uses a monospace font.